manifests/metering-config/azure-blob-storage.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: "operator-metering"
+spec:
+  storage:
+    type: "hive"
+    hive:
+      type: "azure"
+      azure:
+        container: "bucket1"
+        storageAccountName: "meteringazure"
+        secretAccessKey: "your-secret-here"
+        #set to false if you want to reference an exting secret, and set the secretName to the filename
+        createSecret: true
+        secretName: ""
```

Comment:
Set to existing secret name, look at `s3-storage.yaml` for an example. Eg: `secretName: "my-azure-secret"`